examples/ovs-cni.yml

```diff
@@ -14,6 +14,7 @@ spec:
         tier: node
         app: ovs-cni-plugin
     spec:
+      hostNetwork: true
```

Comment:
We should add it to cluster/examples/ too

Reply:
Done